### PR TITLE
Clarify XML Support

### PR DESCRIPTION
**Pull request by os-d**

### CODE REVIEW

File: SetupDataPkg/Docs/ConfigurationFiles/ConfigurationFiles.md [hunk 1]
```diff
@@ -69,16 +70,20 @@ to denote an array ID tag value. [GenCfgData.py](../../Tools/GenCfgData.py) will
 content to backend database and generate the same binary data blob.
 
 - MU version of ConfigEditor supports 3 more option for loading from and saving to SVD (Setup Variable Data) files:
-  - Save Full Config Data to SVD File (supported for YAML/XML/Merged YAML + XML):
-      Saving the entire defined YAML structure into encoded binary settings format. This format is useful when many tags
-      of settings need updating at once, as it will save transmission overhead when serial method is selected. But this
-      will *touch* all configurations defined.
-  - Save Config Changes to SVD File (Supported for YAML):
+  - Save Full Config Data to SVD File:
```
Comment (apop5) [inline SetupDataPkg/Docs/ConfigurationFiles/ConfigurationFiles.md:73]: I think this section can be made clearer, and the recommended methods can be specified here.

we have 3 classes of file operations:

Var List Binary
           Save Config Data to Var List Binary
           Load Config Data from Var List Binary

Var List Binaries <expand on usecase here>

SVD File
           Safe Full Config Data to SVD File
           Save Config Changes to SVD File
           Load Config from SVD File

SVD files are used in conjunction with the ConfigApp that is part of the SetupDataPkg. 
<expand on usecase>

Change File
           Safe Full Config Data to Change File
           Save Config Changes to Change File
           Load Config Changes from Change file
<expand on usecase here>

File: SetupDataPkg/Docs/Overview/Overview.md [hunk 2]
```diff
@@ -162,32 +163,43 @@ firmware volume.
...
 
 ![UEFI Build](Images/uefi_boot_flow.png)
 
+### Settings Update Boot Flow
+
 - Formatted update configuration data from USB, serial port or OS application will first be stored to UEFI variable storage,
 followed by a system reboot.
-- Upon next boot, stored formatted variable will be authenticated and decoded by DFCI framework. Decoded configuration
-data will be dispatched to corresponding platform authored setting provider.
+- Upon next boot, stored formatted variable will be authenticated (if the
```
Comment (apop5) [inline SetupDataPkg/Docs/Overview/Overview.md:197]: Could "Upon next boot" be changed to "Upon the reboot"?

"stored formatted variable" would be clearer as "formatted configuration data from UEFI Variable storage"

File: SetupDataPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md [hunk 3]
```diff
@@ -149,9 +153,118 @@ will omit the integration steps for these features. For more information about D
...
 ``` bash
 [FV.YOUR_DXE_FV]
   # INF MdeModulePkg/Application/UiApp/UiApp.inf
 ```
+
+## Profiles Integration
+
+In order to use configuration profiles, the platform must include the above changes as well as include a YAML file that
+contains the default values for the generic profile. If additional profiles are required, the platform must include YAML
+delta files (.dlt) for each profile that are overrides on top of the generic profile.
```
Comment (apop5) [inline SetupDataPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md:167]: should mention how the .dlt files are created here through ConfigEditor "Save Config Changes to Change File" vs "Save Full Config Changes to Change FIle".

.dlt is doens't need to be mentioned, as it is necessary to be named that.

File: SetupDataPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md [hunk 4]
```diff
@@ -149,9 +153,118 @@ will omit the integration steps for these features. For more information about D
...
+  self.env.SetValue("DELTA_CONF_POLICY", self.mws.join(self.ws, "PlatformPkg", "CfgData", "Profile1.dlt") + ";" +\
+                    self.mws.join(self.ws, "PlatformPkg", "CfgData", "Profile2.dlt") + ";" +\
+                    ...
+                    self.mws.join(self.ws, "PlatformPkg", "CfgData", "ProfileN.dlt"), "Platform Hardcoded")
+  ...
+```
+
+### Platform DEC Changes
+
+The platform must define file GUIDs for each additional profile beyond the generic profile (which has a defined GUID
```
Comment (apop5) [inline SetupDataPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md:187]: "file GUIDs" should just be "Guids".

File: SetupDataPkg/Docs/PlatformIntegration/SoftwareComponentOverview.md [hunk 5]
```diff
@@ -22,6 +24,7 @@ For more general background about the steps necessary to integrate the configura
 | Driver | Location |
 | ---| ---|
 | ConfDataSettingProvider | SetupDataPkg/ConfDataSettingProvider/ConfDataSettingProvider.inf |
+| ConfProfileMgrDxe | SetupDataPkg/ConfProfileMgrDxe/ConfProfileMgrDxe.inf |
 
 ## UEFI Applications
```
Comment (apop5) [inline SetupDataPkg/Docs/PlatformIntegration/SoftwareComponentOverview.md:30]: Missing ActiveProfileSelectorLibNull lib.

File: SetupDataPkg/Docs/Profiles/Overview.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+| SetupVariable Flow | [Link](../Overview/Overview.md) |
+| Configuration YAML Spec | <https://slimbootloader.github.io/specs/config.html#configuration-description-yaml-explained> |
+| | |
+
+## Introduction
+
+### Configuration Profiles
+
+UEFI Configuration Profiles, historically called BIOS Flavors, are sets of defined values for UEFI configuration variables.
+Such profiles are useful in the datacenter case where different owners may use the same hardware but have different
```
Comment (apop5) [inline SetupDataPkg/Docs/Profiles/Overview.md:44]: we can drop data center. Its useful in other scenarios too.

File: SetupDataPkg/Docs/PlatformIntegration/SoftwareComponentOverview.md (hunk 5 above)
Comment (kuqin12) [reply SetupDataPkg/Docs/PlatformIntegration/SoftwareComponentOverview.md:30]: Probably make that as a separate library section.

**Review by kuqin12 — APPROVED** [review]

**Review by apop5 — APPROVED** [review]

File: SetupDataPkg/Docs/ConfigurationFiles/ConfigurationFiles.md (hunk 1 above)
Comment (os-d) [reply SetupDataPkg/Docs/ConfigurationFiles/ConfigurationFiles.md:73]: Sure, I'll add and clarify further

File: SetupDataPkg/Docs/Overview/Overview.md (hunk 2 above)
Comment (os-d) [reply SetupDataPkg/Docs/Overview/Overview.md:197]: Sure, will change.

File: SetupDataPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md (hunk 3 above)
Comment (os-d) [reply SetupDataPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md:167]: Thanks, will add.

File: SetupDataPkg/Docs/PlatformIntegration/SoftwareComponentOverview.md (hunk 5 above)
Comment (os-d) [reply SetupDataPkg/Docs/PlatformIntegration/SoftwareComponentOverview.md:30]: Will add.

**Review by apop5 — APPROVED** [review]

**Pull request merged**